charts/ia-case-notifications-api/values.yaml

```diff
@@ -41,6 +41,7 @@ java:
         - admin-officer-review-hearing-requirements-email
         - ctsc-admin-ftpa-submitted-email
         - respondent-ftpa-submitted-email
+        - customer-services-email
```

Comment:
do you need `customer-services-telephone` too?

Reply:
Yes I do, thanks for that.